src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url("../images/star-active.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:last-child {
+  margin-right: 4px;
+  background-image: url("../images/star.svg")
+  }
```

Comment:
you need to reuse all styles from the stars task, we should have the opportunity to change the number of cells by changing the modifier